src/main/java/org/traccar/protocol/NavtelecomProtocolDecoder.java

```diff
@@ -196,9 +196,10 @@ protected Object decode(
                     for (int j = 0; j < bits.length(); j++) {
                         if (bits.get(j)) {
 
+                            int bitNum = j + 1;
```

Comment:
Can we just have `j` value to be `1` to `bits.length()`?

Reply:
Anton, most likely I don’t understand the translation exactly, can you formulate the question differently? Sorry)

Reply:
My point is that you should just have `j` start from 1 instead of 0. That way you won't need a second variable.

Reply:
Great solution.  I'll fix it today and test it again.